README.md

```diff
@@ -65,9 +65,15 @@ As mentioned before, when it comes to running the tests, you've got 2 options:
 * Running the tests on an existing cluster using the `driver` along with the provided automation
 
 NOTE: Depending on the version of Charmed Kubeflow you want to test, make sure to checkout to the appropriate branch with `git checkout`:
+- Charmed Kubeflow 1.9 -> `track/1.9`
```

Comment:
If we branch off after the merge, we can remove this and bundle it together with the branching off for track/1.9

Reply:
I think it's good to keep it here, since I imagine a lot of users view the README from the `main` branch.

Reply:
I was thinking of the option to add something for latest/edge. On one hand, this is not a version we support. On the other hand, it's not documented anywhere what to use (even for us) if you want to test `latest/edge`.

Reply:
I can add a small sentence